File: Postman/Postman-examples/uco-api-validate-example-device-url.py
import requests
import json

url = "http://0.0.0.0:8080/shacl/UCO/api/validate"

payload = json.dumps({
  "validationType": "large",
  "embeddingMethod": "URL",
  "contentToValidate": "https://raw.githubusercontent.com/casework/CASE-Examples/master/examples/illustrations/device/device.json",
  "contentSyntax": "application/ld+json",
  "reportSyntax": "application/ld+json"
})
headers = {
  'Accept': 'application/json',
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)

File: Postman/Postman-examples/uco-api-validate-example-device-url.java
Unirest.setTimeouts(0, 0);
HttpResponse<String> response = Unirest.post("http://0.0.0.0:8080/shacl/UCO/api/validate")
  .header("Accept", "application/json")
  .header("Content-Type", "application/json")
  .body("{\n    \"validationType\": \"large\",\n    \"embeddingMethod\": \"URL\",\n    \"contentToValidate\": \"https://raw.githubusercontent.com/casework/CASE-Examples/master/examples/illustrations/device/device.json\",\n    \"contentSyntax\": \"application/ld+json\",\n    \"reportSyntax\": \"application/ld+json\"\n}")
  .asString();

